.github/workflows/CI.yml

```
@@ -50,15 +56,14 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        gcc-ver: [6, 7, 8]
+        gcc-ver: [7, 8, 9, 10]
```

I'd rather keep testing on the oldest supported compiler until I explicitly decide to break backwards compatibility.

```suggestion
        gcc-ver: [6, 7, 8, 9, 10]
```